.github/workflows/ci-generated.yml

```diff
@@ -18,6 +18,31 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: "3.7"
+    - name: Setup Git Profile
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+    - name: Revert Previous Change and Merge Master
+      run: |
+        git ls-remote origin
+        if git ls-remote origin | grep refs/ci_generated/master; then
+          # If the remote hidden branch exists
+          # Map the remote ref to branch.
+          git fetch origin +refs/ci_generated/master:refs/remotes/origin/ci_generated/master
+          git checkout -b ci_generated-prev origin/ci_generated/master
+
+          # Revert the previous generated files, so that the merge is clean.
+          git revert HEAD
```

Comment:
This should not longer be needed with `git merge -s ours` as this should already discard everything from the previous `ci_generated` branch.